etc/vendor-libmongoc.sh

```diff
@@ -130,16 +132,9 @@ echo "RENAMING header files"
   popd
 )
 
-# Here we apply a number of larger patches that don't fit into a single sed line, specifically
-# an inability to use the `inttypes.h` header in an umbrella header file due to limitations with
-# how clang is building the module
-echo "PATCHING libmongoc"
-git apply "${ETC_DIR}/inttypes-non-modular-header-workaround.diff"
-
-# These patches are temporary workarounds to give us early access to the directConnection URI option and a bug fix.
-# This should be removed from the script when we update our vendored libmongoc to 1.17 via SWIFT-766.
-git apply "${ETC_DIR}/CDRIVER-3532-directConnection.diff"
-git apply "${ETC_DIR}/CDRIVER-3623-fix-setting-apm-in-pooled-client.diff"
+# Here we would apply any number of larger patches that don't fit into a single sed line.
+# echo "PATCHING libmongoc"
```

Comment:
looks like the `echo` was left commented out

Reply:
fixed